NeuroID/src/main/java/com/neuroid/tracker/utils/NIDMetaData.kt

```diff
@@ -82,6 +92,65 @@ class NIDMetaData(context: Context) {
         }
     }
 
+    internal fun getLocation(context: Context) {
+        if (ActivityCompat.checkSelfPermission(
+                context,
+                Manifest.permission.ACCESS_FINE_LOCATION
+            ) != PackageManager.PERMISSION_GRANTED || ActivityCompat.checkSelfPermission(
+                context,
+                Manifest.permission.ACCESS_COARSE_LOCATION
+            ) != PackageManager.PERMISSION_GRANTED
+        ) {
+            // need to request the permissions, return
```

Comment:
should we return a NIDLocation obj with authorization set as "denied" since we will know at this point?

Reply:
Yeah I can do this as well.